hack/make-rules/test-e2e-node.sh

```diff
@@ -119,7 +119,7 @@ if [ "${remote}" = true ] && [ "${remote_mode}" = gce ] ; then
     gci_image=$(gcloud compute images list --project "${image_project}" \
     --no-standard-images --filter="name ~ 'cos-beta.*'" --format="table[no-heading](name)")
     images=${gci_image}
-    metadata="user-data<${KUBE_ROOT}/test/e2e_node/jenkins/gci-init.yaml,gci-update-strategy=update_disabled"
```

Comment:
Are you moving the `./test/e2e_node/jenkins/gci-init.yaml` to the new location or making a copy?

Reply:
This change copies `test/e2e_node/jenkins/gci-init.yaml` to `test/e2e_node/cos-init/gci-init.yaml` file. We need follow-up change in `test-infra` repo to read config files from `test/e2e_node/cos-init` directory and then we can delete the `jenkins` directory.

Unfortunately, `git` doesn't capture the files that are copied.